### PR TITLE
fix: Bring back @talend libs in TALEND_APP_INFO

### PR DESCRIPTION
**Pull request by romainseb**
**What is the problem this PR is trying to solve?**
`TALEND_APP_INFO` used to have all @talend applications informations
We only have the libs coming from the CDN right now
We use it into the About Dialog into Data Inventory, we would like to have it back :)

Before the CDN change :
https://github.com/Talend/ui-scripts/blame/ab17e60e2fe78d6f6928ea84c03297a0a275677e/packages/config-react-webpack/config/webpack.config.js#L110

**What is the chosen solution to this problem?**
Add the talend lib versions (and keep the cdn for the meta elements) 

**Please check if the PR fulfills these requirements**

- [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: tools/scripts-config-react-webpack/config/webpack.config.js [hunk 1]
```diff
@@ -93,28 +96,92 @@ function getSassLoaders(enableModules, sassData, mode) {
...
+					acc[name] = info.version;
+				}
+				return acc;
+			}, talendLibraries);
+	} else if (fs.existsSync(packagelockPath)) {
+		// eslint-disable-next-line
+		const packageLock = require(packagelockPath);
+		Object.keys(packageLock.dependencies)
+			.filter(k => k.startsWith('@talend'))
+			.reduce((acc, key) => {
```
Comment (gjou-tlnd) [inline tools/scripts-config-react-webpack/config/webpack.config.js:154]: this won't work with npm > 6 (which is used on TDP)
also, it's weird webpack doesn't have a plugin for this 🤔

Comment (romainseb) [reply tools/scripts-config-react-webpack/config/webpack.config.js:154]: Thanks for the feedback 👍🏼 
Is it ok to only check package-lock for npm 6+ or do i have to handle the version before ?

Comment (gjou-tlnd) [reply tools/scripts-config-react-webpack/config/webpack.config.js:154]: I guess you would be better off getting versions from `node_modules/@talend/**/package.json`

**Review by jmfrancois — APPROVED** [review]
This code will not give you only deps but also dev deps.
Lets iterate on the issue !

**Review by jmfrancois — APPROVED** [review]
au top !

File: tools/scripts-config-react-webpack/config/webpack.config.js [hunk 2]
```diff
@@ -93,31 +98,94 @@ function getSassLoaders(enableModules, sassData, mode) {
...
 	};
 }
 
+function getVersions() {
+	const talendLibraries = cdn
+		.getModulesFromLockFile()
+		.filter(Boolean)
+		.map(info => {
+			return { version: info.version, name: info.name };
+		});
```
Comment (ncomont) [inline tools/scripts-config-react-webpack/config/webpack.config.js:178]:
```suggestion
		.map(info => ({ version: info.version, name: info.name }));
```

**Pull request merged**